docker/Dockerfile

```diff
@@ -13,7 +13,7 @@
 FROM archlinux:base-20210829.0.32635
 
 # ↓↓↓ UPDATE MIRRORS HERE ↓↓↓
-RUN echo -e "Server=http://host.docker.internal:7878/\$repo/os/\$arch\nServer=https://archive.archlinux.org/repos/2021/08/29/\$repo/os/\$arch" | tee /etc/pacman.d/mirrorlist
```

Comment:
Why are we removing the local cache?

Reply:
The local cache only worked if you had the cache container running. That was only ever done on the iMacs when they were bamboo runners. They aren't that any more

Reply:
:+1:

Reply:
If you want another option and have a computer somewhere to run something on in the lab I can give you another caching solution